openshift/init-app.sh

```diff
@@ -47,6 +47,7 @@ OPENSHIFT_USER='system:admin'
 OPENSHIFT_TEMPLATE_PATH='openshift/koku-template.yaml'
 CODE_REPO='https://github.com/project-koku/koku.git'
 REPO_BRANCH='master'
+EMAIL_SERVICE_PASSWORD=$EMAIL_SERVICE_PASSWORD
```

Comment:
It might also be good to document that this is required in the docs PR that is out right now, that is if it hasn't already been @chambridge